src/counter.test.js

```diff
@@ -0,0 +1,29 @@
+const { displayMovies } = require('./index.js');
+const { addMoviesCounter } = require('./modules/counter.js');
+
+describe('Items Count', () => {
+  it('No items count', () => {
+    const arr = [];
+
+    displayMovies(arr);
+
+    expect(addMoviesCounter('list-of-shows')).toBe(0);
+  });
+
+  it('Full Items Count', async () => {
+    const arr = [
+      {
+        show: { id: 0, image: { medium: 'src' }, name: 'name' },
+        likes: 0,
+      },
+      {
+        show: { id: 0, image: { medium: 'src' }, name: 'name' },
+        likes: 0,
+      },
+    ];
+
+    await displayMovies(arr);
+
+    expect(addMoviesCounter('ul li')).toBe(2);
+  });
+});
```

Comment:
- [ ] All your tests are failing. Kindly look into that and fix them.

  ![image](https://user-images.githubusercontent.com/54848363/226103571-b2dff59c-da6f-42bd-b5fb-f0e77777c2eb.png)